doc/api/modules.md

```diff
@@ -425,6 +426,30 @@ These are mostly for historic reasons.  **You are highly encouraged
 to place your dependencies locally in `node_modules` folders.**  They
 will be loaded faster, and more reliably.
 
+## The module wrapper
+
+<!-- type=misc -->
+
+Before Node.js hands off your module to be executed by V8, it wraps your code
```

Comment:
@addaleax @evanlucas  Do you have any suggestions for something more appropriate? Some ideas:

- s/V8/the JavaScript [runtime|engine]
- Change the whole sentence to "Before Node.js executes the code in your module, it wraps it in..."